src/main/kotlin/ai/digital/integration/server/common/constant/TerraformConstants.kt

```diff
@@ -0,0 +1,11 @@
+package ai.digital.integration.server.common.constant
+
+object TerraformConstants {
```

Comment:
Please check how extensions work in Gradle.
As now there is still no way how to override those values during the run.

Reply:
Took the example from here: https://github.com/xebialabs/integration-server-gradle-plugin/blob/master/src/main/kotlin/ai/digital/integration/server/deploy/internals/DeployDockerClusterHelper.kt#L100